src/main/impl/Build.cpp

```diff
@@ -36,6 +36,9 @@ char const* const versionString = "1.0.3"
         "-release"
 #endif
 
+#define VAL(CLIO_GIT_COMMIT_HASH) #CLIO_GIT_COMMIT_HASH
+#define TOSTRING(CLIO_GIT_COMMIT_HASH) VAL(CLIO_GIT_COMMIT_HASH)
```

Comment:
Why did we need to add these? Why couldn't we follow the same pattern we were following for debug builds?